fpga_interchange/xc7_device_timing_patching.py

```diff
@@ -0,0 +1,246 @@
+            with open(os.path.join(timing_dir, _file), 'r') as f:
+                tile_data = json.load(f)
+            print(_file, tile_data["tile_type"])
+            if tile_data['tile_type'] not in string_map:
+                continue
+            tile_name = string_map[tile_data['tile_type']]
+            tileType = tile_name_tileType_map[tile_name]
+            for name, data in tile_data['wires'].items():
+                if data is None:
+                    continue
+                wire_name = string_map[name]
+                for wire in tileType_wire_name_wire_list_map[(tileType,
+                                                              wire_name)]:
+                    if wire not in wire_node_map:
+                        continue
+                    node = wire_node_map[wire]
+                    model = node_model_map[node]
+                    model = (model[0] + float(data['cap']) * 1e-9,
+                             model[1] + float(data['res']) * 1e-6)
+                    node_model_map[node] = model
+            for data in tile_data['pips'].values():
+                wire0 = string_map[data['src_wire']]
+                wire1 = string_map[data['dst_wire']]
+                key = (tileType, wire0, wire1)
+                if key not in tileType_wires_pip_map:
+                    continue
+                pip = tileType_wires_pip_map[key]
+                model_values = data['src_to_dst']
+                delays = (0, 0, 0, 0)
+                if model_values['delay'] is not None:
+                    delays = tuple(model_values['delay'])
+                cap = 0
+                if model_values['in_cap'] is not None:
+                    cap = float(model_values['in_cap'])
+                res = 0
+                if model_values['res'] is not None:
+                    res = float(model_values['res'])
+                pip_models[pip] = (cap, res, delays)
+            for site in tile_data['sites']:
+                siteType = siteName_siteType_map[string_map[site['type']]]
+                for sitePin, dic in site['site_pins'].items():
+                    sitePin_obj = siteType_name_sitePin_map[(
+                        siteType, string_map[sitePin])]
+                    if dic is None:
+                        dic = {}
+                        dic['delay'] = [0, 0, 0, 0]
+                    elif 'res' in dic.keys():
+                        sitePin_obj.model.init('resistance')
+                        sitePin_obj.model.resistance.slow.init('slow')
+                        sitePin_obj.model.resistance.slow.slow.typ.typ = float(
+                            dic['res']) * 1e-6
+                    else:
+                        sitePin_obj.model.init('capacitance')
+                        sitePin_obj.model.capacitance.slow.init('slow')
+                        sitePin_obj.model.capacitance.slow.slow.typ.typ = float(
+                            dic['cap']) * 1e-6
```

Comment:
capacitance is being multiplied by `1e-9` in other places, are you sure the `1e-6` here is correct?

Reply:
I missed that, thank you for point it out